### DISCUSSION TITLE
How can style disabled React-Date-Picker component in dark mode?

### DISCUSSION THREAD

**Discussion by veera-selvam-s** [wojtekmaj/react-date-picker#493] [Q&A]
I have tried to style the component when it gets disable and also in dark mode . Im using next js for theme change. But I can't achieve it . can anyone please help me?

**Comment by wojtekmaj** [comment]
That question is too generic to answer. I don't know how you provide theme to your components and what styling system are you using.

One of the approaches you can take is passing different class names to React-Date-Picker depending on the theme selected.
